README.md

```diff
@@ -109,7 +109,7 @@ open-attestation filter examples/wrapped-documents/example.0.json tmp/example.0.
 ✔  success  Obfuscated document saved to: tmp/example.0.out.json
 ```
 
-## Encrypting document
+## Decrypting document
```

Comment:
I think the title got mixed up, this one is encrypt the other is decrypt.

Out of curiosity why do you use the cli to decrypt document for ?

Reply:
Ah when downloading from the endpoints. I'm foreseeing that people will ask for this when they have the encrypted payload link for notarize.